frontend/packages/ceph-storage-plugin/src/components/modals/block-pool-modal/create-block-pool-modal.tsx

```diff
@@ -69,8 +69,19 @@ export const CreateBlockPoolModal = withHandlePromise((props: CreateBlockPoolMod
         setIsSubmit(false);
         clearTimeout(timer);
         onPoolCreation(state.poolName);
-      } else if (newPoolLoaded && newPool?.status?.phase === POOL_STATE.FAILED) {
+      } else if (
+        newPoolLoaded &&
+        (newPool?.status?.phase === POOL_STATE.RECONCILE_FAILED ||
+          newPool?.status?.phase === POOL_STATE.FAILURE)
+      ) {
         dispatch({ type: BlockPoolActionType.SET_POOL_STATUS, payload: POOL_PROGRESS.FAILED });
+        dispatch({
+          type: BlockPoolActionType.SET_ERROR_MESSAGE,
+          payload: t('ceph-storage-plugin~Pool {name} got created with errors').replace(
+            '{name}',
+            state.poolName,
+          ),
```

Comment:
Instead of this we want to add it in `packages/ceph-storage-plugin/src/utils/block-pool.tsx` -> PROGRESS_STATUS an entry of error message.
The <BlockPoolStatus> component already handles the `{name}` replacement.
```
      <EmptyStateBody data-test="empty-state-body">
          {error ? getErrorMessage(error) : statusObj.desc.replace('{name}', name)}
        </EmptyStateBody>
 ```
 https://github.com/openshift/console/blob/40a20977d27410c4361ebc3b43b67464a2e7a434/frontend/packages/ceph-storage-plugin/src/components/block-pool/body.tsx#L38-L47